components/HomepageSection/UseCases.jsx

```diff
@@ -105,6 +105,18 @@ const UseCases = () => (
           {...item}
         />
       ))}
+
+    </div>
+
+    <div>
+      <Button
+        variant="outline"
+        size="xl"
+        asChild
+        className="mt-12 max-sm:mt-3 col-start-2 mx-auto"
+      >
+        <Link href="/explore#use-cases">See more use cases</Link>
```

Comment:
@atepem can you please confirm with Roman that it should open `/explore#use-cases` and not just `/explore`?

Reply:
Roman's said that he'd rather it go to /explore instead of the specific section so I can change that now :)